docs/user-guide/model.rst

```diff
@@ -5,7 +5,7 @@ Data Model
 Felis's `data model <../dev/internals.html#module-felis.datamodel>`__ is defined by a set of
 `Pydantic <https://docs.pydantic.dev/latest/>`__ classes which describe the semantics of tabular data.
 In addition to the standard conceptual constructs of a relational database schema, such as tables and columns,
-Felis provides a way to attach extra metadata like units of measurement on columns.
+Felis provides a way to attach extra metadata to these elements like units of measurement on columns.
```

Comment:
comma after "elements"

Reply:
> comma after "elements"

Missed this, but will incorporate it into a user branch with additional small fixes if I find any other grammar issues, etc.